plugins/woocommerce/templates/myaccount/orders.php

```diff
@@ -68,7 +68,7 @@
 
 								if ( ! empty( $actions ) ) {
 									foreach ( $actions as $key => $action ) { // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
-										echo '<a href="' . esc_url( $action['url'] ) . '" class="woocommerce-button' . esc_attr( $wp_button_class ) . ' button ' . sanitize_html_class( $key ) . '">' . esc_html( $action['name'] ) . '</a>';
+										echo '<a href="' . esc_url( $action['url'] ) . '" class="woocommerce-button' . esc_attr( $wp_button_class ) . ' button ' . sanitize_html_class( $key ) . '" aria-label="' . esc_attr( sprintf( __( 'View order number %s', 'woocommerce' ), $order->get_order_number() ) ) . '">' . esc_html( $action['name'] ) . '</a>';
```

Comment:
We'll need a translators comment here since there's a placeholder, see https://github.com/woocommerce/woocommerce/actions/runs/9896640866/job/27339537309?pr=49424#step:4:27

Reply:
Thanks @dkotter I have added in the translator comment.

Reply:
I don't think this translator comment is correct. The string is `View order number %s` which contains one placeholder, so it should read:

```
/* translators: %s: order number */
```